webkit2/webkit2.init.lisp

```diff
@@ -26,26 +26,31 @@
 
 (defcfun "webkit_get_major_version" :int)
 (defcfun "webkit_get_minor_version" :int)
+(defcfun "webkit_get_micro_version" :int)
 
 (eval-when (:load-toplevel :execute)
   (with-standard-io-syntax
-    (let* ((version (format nil "~a.~a"
-                            (webkit-get-major-version)
-                            (webkit-get-minor-version)))
-           (versym (intern (format nil "WEBKIT2-~a" version)
-                           :keyword)))
+    (let* ((major (webkit-get-major-version))
+           (minor (webkit-get-minor-version))
+           (micro (webkit-get-micro-version))
+           (version (format nil "~a.~a.~a" major minor micro)))
       (when (uiop:version<= "2.26" version)
         ;; Sandboxing only works on Linux as of 2.28.  See
         ;; https://webkitgtk.org/reference/webkit2gtk/unstable/WebKitWebContext.html.
         ;; TODO: Only set feature on Linux?
         (dolist (sym '("WEBKIT2-SANDBOXING" "WEBKIT2-MEDIA" "WEBKIT2-EMOJI"))
-          (pushnew (intern sym :keyword) *features*)))
+          (pushnew (alexandria:make-keyword sym) *features*)))
       (when (uiop:version<= "2.30" version)
         (dolist (sym '("WEBKIT2-MUTE" "WEBKIT2-TRACKING" "WEBKIT2-PASTE-PLAINTEXT"))
-          (pushnew (intern sym :keyword) *features*)))
+          (pushnew (alexandria:make-keyword sym) *features*)))
       (when (uiop:version<= "2.34" version)
         (dolist (sym '("WEBKIT2-CORS-ALLOWLIST"))
-          (pushnew (intern sym :keyword) *features*)))
-      (pushnew versym *features*)))
-
+          (pushnew (alexandria:make-keyword sym) *features*)))
+      (pushnew (alexandria:make-keyword (format nil "WEBKIT-~a" major))
+               *features*)
+      (pushnew (alexandria:make-keyword (format nil "WEBKIT-~a.~a" major minor))
+               *features*)
+      (pushnew (alexandria:make-keyword (format nil "WEBKIT-~a.~a.~a" major minor micro))
+               *features*)))
+  ;; To be deprecated when WebKit3 is released.
```

Comment:
As suggested [in this comment](https://github.com/joachifm/cl-webkit/pull/71#issuecomment-1527945081).